src/settings/validationSettings.ts

```diff
@@ -1,2 +1,19 @@
-export const VENUE_NAME_MAX_CHAR_COUNT = 50;
-export const VENUE_NAME_MIN_CHAR_COUNT = 3;
+import { VenuePlacement } from "types/venues";
+
+import { PLAYA_HEIGHT, PLAYA_VENUE_SIZE, PLAYA_WIDTH } from "./playaSettings";
+
+export const COMMON_NAME_MAX_CHAR_COUNT = 50;
+export const COMMON_NAME_MIN_CHAR_COUNT = 3;
+
+export const COMMON_STRING_MIN_CHAR_COUNT = 3;
+export const COMMON_STRING_MIN_CHAR_COUNT_RE = new RegExp(
+  `.{${COMMON_STRING_MIN_CHAR_COUNT},}`
+);
+
+export const START_DATE_FORMAT_RE = /\d{4}-\d{2}-\d{2}/;
+
+export const DEFAULT_MAP_ICON_PLACEMENT: Omit<VenuePlacement, "state"> = {
+  x: (PLAYA_WIDTH - PLAYA_VENUE_SIZE) / 2,
```

Comment:
should these map settings be imported from the `playa` named file? If those settings are shared between all `map` like templates we should move them to general settings?

Reply:
I checked, they aren't really. Could've just as easily put them in `validationSettings.ts` but I figured it would be easier to just delete the file later once playa settings aren't necessary.

Reply:
To be more clear: most of those playa settings can be marked as deprecated and/or outright deleted. But I didn't want to spend the time to get the all clear on this PR that already changes 22 files.

Still more schemas are around the project, not inside `./src/forms` and marked in https://github.com/sparkletown/internal-sparkle-issues/issues/1462 so these playa settings can also go there.